colibre/auto_plotter/metallicity.yml

```diff
@@ -555,10 +556,11 @@ stellar_mass_star_metallicity_30:
     - filename: GalaxyStellarMassStellarMetallicity/Gallazzi2005_Data.hdf5
     - filename: GalaxyStellarMassStellarMetallicity/Kirby2013_Data.hdf5
     - filename: GalaxyStellarMassStellarMetallicity/Fraser-McKelvie_2021.hdf5
-    - filename: GalaxyStellarMassStellarMetallicity/FIREbox.hdf5
```

Comment:
Same comment regarding the FIREbox data as above.

Reply:
Oh sorry, I took it out and forgot to put it back in.